nix/no-grub.nix

```diff
@@ -0,0 +1,10 @@
+{lib, ...}:{
```

Comment:
Can be removed now. I called `no-bootloader.nix`.

Reply:
https://github.com/nix-community/nixos-images/pull/211/commits/c54ae6f4d1ed6303f33b61f39a0fef9d6c58bff8